actors/account/build.rs

```diff
@@ -5,8 +5,8 @@ fn main() {
         .import_memory()
```

Comment:
i think all the `import_memory` should be removed, otherwise the generated wasm would ask for importing `env::memory` instead of exporting memory module, like:
`(import "env" "memory" (memory (;0;) 16))` vs `(export "memory" (memory 0))`